soda_machine.rb

```diff
@@ -1,18 +1,27 @@
 class SodaMachine
-  attr_reader :sodas, :cash
+  attr_reader :sodas
+  attr_accessor :cash
```

Comment:
cash does not need to be an accessor - action taken before conditional statement was properly formed...